src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -0,0 +1,3710 @@
+openapi: 3.0.1
```

Comment:
Is there a reason this needed to change directories? Could we just keep the openapi.yaml in its original location? (/src/main/resources/data-repository-openapi.yaml)

Reply:
From our conversation: This moves the openapi.yaml into its own directory, which will be helpful when we're serving this to the new swagger ui.